include/net/nrf_cloud.h

```diff
@@ -107,7 +107,9 @@ enum nrf_cloud_evt_type {
 	 *  be populated with a @ref nrf_cloud_disconnect_status value.
 	 */
 	NRF_CLOUD_EVT_TRANSPORT_DISCONNECTED,
-	/** A FOTA update has started. */
+	/** A FOTA update has started.
+	 * This event is only sent if @kconfig{CONFIG_NRF_CLOUD_FOTA_AUTO_START_JOB} is enabled.
```

Comment:
Just commenting. I thought we need the normal brackets () instead of the curly ones, but the curly ones seem to work fine, too.

Reply:
normal brackets resulted in a doc build failure.  other NCS files use curly.

Reply:
You are so right. I misread that in my earlier chat with a developer who was asking about that. Or actually, I should get new glasses. He had it right with the curly ones.